content/applications/hr/payroll.rst

```diff
@@ -332,23 +345,24 @@ Personal Info`.
    :align: center
    :alt: Personal information that appear on employee cards to enter.
 
-To edit an entry, select it from the list. Then, click the :guilabel:`Edit` button, and modify the
-entry. When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel
-the edits.
+To edit an entry, select it from the list, and modify the entry. To make a new entry, click the
```

Comment:
line 341 add in section

Reply:
After making so many changes, I think this section is for the personal info section -  if so, I don't think it needs sections since it calls out the required info, and the information seems understandable and clear to me